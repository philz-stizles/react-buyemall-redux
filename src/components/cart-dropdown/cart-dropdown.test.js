import { shallow } from 'enzyme'
import React from 'react'
import CartDropdown from './cart-dropdown.component'

it('to render a CartDropdown componenet', () => {
    expect(shallow(<CartDropdown />)).toMatchSnapshot()
})