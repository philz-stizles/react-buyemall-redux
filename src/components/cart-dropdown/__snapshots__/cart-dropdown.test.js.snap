// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`to render a CartDropdown componenet 1`] = `ShallowWrapper {}`;
